source/getting-started/routes-and-templates.md

```diff
@@ -0,0 +1,166 @@
+To demonstrate the basic setup and processing of an Ember application, the next few lessons are going to walkthrough building a very simple application for a property rental site called Super Rentals.  It will start with just a homepage, an about page and a contact page.  Let's take a look at the application from the user perspective before we get started.
+
+![super rentals homepage screenshot](../../images/routes-and-templates/ember-super-rentals-index.png)
+
+We arrive at the home page which shows a list of rentals.  From here, we will be able to navigate to an about page and a contact page.  
+
+We already have the new app called super-rentals created. Before we start building the three pages for our app, we are going to clear out the contents of the `app/templates/application.hbs` file and only leave the {{outlet}} code in place.  We'll talk more about the role of the `application.hbs` file after our site has a few pages.
```

Comment:
Where was this created? If I'm starting here, and I didn't create this app, where should I go to create it?